Integrations/FeedOffice365/FeedOffice365.yml

```diff
@@ -112,30 +108,24 @@ script:
       secret: false
     - auto: PREDEFINED
       default: false
-      defaultValue: 'IPS'
-      description: The indicator type. Can be "IPS", "URLS", or "Both". The default value is "IPS".
+      defaultValue: IPs
+      description: The indicator type. Can be "IPs", "URLs", or "Both". The default
+        value is "IPs".
       isArray: false
       name: indicator_type
       predefined:
-      - 'IPS'
-      - 'URLS'
-      - 'Both'
+      - IPs
+      - URLs
+      - Both
       required: false
       secret: false
     deprecated: false
     description: Gets indicators from the feed.
     execution: false
-    name: get-indicators
-    outputs:
-    - contextPath: O365.Indicator.Value
-      description: The indicator value.
-      type: String
-    - contextPath: O365.Indicator.Type
-      description: The indicator type.
-      type: Unknown
+    name: office365-get-indicators
   dockerimage: demisto/python3:3.7.5.4328
```

Comment:
You'll need to update the docker image to the latest tag version. You'll see - the `demisto-sdk` will complain and tell you.